packages/upset/src/components/ElementView/ElementTable.tsx

```diff
@@ -40,10 +41,16 @@ function useColumns(columns: string[]) {
  * Table to display elements
  */
 export const ElementTable: FC = () => {
-  const attributeColumns = useRecoilValue(attributeAtom);
+  const attColumns = useRecoilValue(attributeAtom);
```

Comment:
I prefer that this is fully spelled out. Makes things more readable (for me who never works in the code)

Reply:
Agree with jack here